libraries/SITL/SIM_last_letter.h

```diff
@@ -41,7 +41,7 @@ class last_letter : public Aircraft {
     }
 
 private:
-    static const uint16_t fdm_port = 9002;
+    static const uint16_t fdm_port = 5002;
```

Comment:
This is a big question mark for me.
Originally I used the 9002/3 ports for SITL-simulator communication.
Afterwards, I noticed that Gazebo-SITL uses 5002.
On the [SITL architecture page](http://ardupilot.org/dev/docs/sitl-simulator-software-in-the-loop.html) I see that 5502/3 is the proposed pair.

Is there a preferred/proposed pair of ports? Is there a developer recommendation?